underscore.js

```diff
@@ -436,6 +436,18 @@
   _.toArray = function(obj) {
     if (!obj) return [];
     if (_.isArray(obj)) return slice.call(obj);
+    if (_.isString(obj)) {
+      // Keep surrogate pair characters together
+      // https://mathiasbynens.be/notes/javascript-unicode
+      return _.reduce(obj, function (output, char, idx) {
+        if (idx > 0 && obj[idx-1].match(/[\uD800-\uDBFF]/) && char.match(/[\uDC00-\uDFFF]/)) {
```

Comment:
I'd prefer a lookahead to a lookbehind here, but that would require you to switch this up from a reduce to a simple for loop. At the very least, I'd use `<` comparison with the actual boundary characters (`x < "\uD800"`, etc.) instead of using a regexp.